### ISSUE TITLE
Is Airpal deprecated? 

### ISSUE THREAD

**Issue by chethanuk** [airbnb/airpal#233]
Since there is not being developed in this repo...

Wanted to ask if the project is no longer used...

**Comment by kbill4345** [comment]
i'm have the same issues.

**Comment by chethanuk** [comment]
> i'm have the same issues.

Last [commit](https://github.com/airbnb/airpal/commit/e3e65283d66d866c3321fd921e02428c6aed1747) is from 2016 so you can assume so..

**Comment by snsarma** [comment]
@ChethanUK Are you able to run the airpal application locally ? I am getting stuck after the first few steps and I am unable to move past them. Inspite of installing gradle and setting the appropriate system environment variables.

**Comment by chethanuk** [comment]
No, we had moved to [quix](https://github.com/wix/quix) instead
